newIDE/app/src/InAppTutorial/InAppTutorialProvider.js

```diff
@@ -126,7 +126,10 @@ const InAppTutorialProvider = (props: Props) => {
         tutorialId: inAppTutorial.id,
         initialProjectData: inAppTutorial.initialProjectData || {},
         initialStepIndex: 0,
-        inAppTutorial,
+        inAppTutorial: {
+          ...inAppTutorial,
+          isMiniTutorial: true, // Force to true to avoid saving warning being triggered.
```

Comment:
this is to ease testing when loading a JSON and avoid having a warning "You're leaving without save" even when it's finished